tests/python/pytorch/graphbolt/test_subgraph_sampler.py

```diff
@@ -0,0 +1,21 @@
+import dgl
+import dgl.graphbolt
+import torch
+import torchdata.datapipes as dp
+
+
+def test_SubgraphSampler():
+    graph = dgl.add_reverse_edges(dgl.rand_graph(20, 60))
```

Comment:
Can you try with https://github.com/dmlc/dgl/blob/master/python/dgl/graphbolt/graph_storage/csc_sampling_graph.py?

Reply:
How should I generate a random undirected graph with `CSCSamplingGraph`?

Reply:
I'm using scipy now.